sdk/cognitivelanguage/ai-language-text/src/models.ts

```diff
@@ -521,6 +583,10 @@ export interface HealthcareEntityRelation {
    * The list of healthcare entities and their roles in the healthcare relation.
    */
   readonly roles: HealthcareEntityRelationRole[];
+  /**
+   * The confidence score between 0 and 1 of the extracted relation.
+   */
+  readonly confidenceScore?: number;
```

Comment:
`?:` in typescript means optional, Is confidence score could be nullable in here?

Reply:
Yes because it is not supported in v2022-05-01.